isBinary_Search_Tree/isbst.cpp

```diff
@@ -0,0 +1,52 @@
+#include <bits/stdc++.h>
+using namespace std; 
+  
+struct Node                                   //structure of the tree node 
+{ 
+    int data; 
+    struct Node* left;
+    struct Node* right; 
+}; 
+  
+struct Node* createNode(int data)            //create a new node to the tree function
+{ 
+    struct Node* node = new Node;  
+    node->left = NULL;
+    node->right = NULL; 
+    node->data = data;
+    return (node); 
+} 
+
+bool isbst(Node* root, int min, int max)   //function to check if tree with root node is bst or not
+{ 
+    if (root == NULL)                      //empty tree is a bst
+        return true; 
+ 
+    if(root->data < min || root->data >max) return false;
```

Comment:
Please fix the indentation.

Reply:
Also, have the comments about the function, above its definition for better readability. Try following this with other general comments too.

Reply:
Surely.